modules/tinymce/src/core/test/ts/module/McAgar.ts

```diff
@@ -1,7 +1,10 @@
+/* eslint-disable @tinymce/no-main-module-imports */
 /*
   This is a helper module that allows us to ensure that TinyMCE core is included/compiled
   before mcagar loads. This ensures that the core of the editor isn't loaded from pre-built
   files in node_modules and is always the latest version.
  */
 import 'tinymce';
 export * from '@ephox/mcagar';
+
+import 'tinymce/themes/silver/Main';
```

Comment:
```suggestion
import 'tinymce/themes/silver/Main';
export * from '@ephox/mcagar';
```
This should probably be registered before loading the mcagar components. Overall I don't think it'll matter, but this makes more logical sense as it's setting up TinyMCE and then exporting the components to interact with it.

Reply:
Oh interesting. The tests pass (both here and on the model branch), but that makes sense.